app/schemas/project_database.py

```diff
@@ -1,6 +1,7 @@
-from marshmallow import Schema, fields, validate, pre_load
+from marshmallow import Schema, fields, validate, pre_load, ValidationError, validates
```

Comment:
Hey @Masozera , do you mind removing these unused imports `ValidationError` and `validates`?

Reply:
Alright